Libplanet/Blocks/Block.cs

```diff
@@ -150,6 +150,52 @@ public byte[] ToBencodex(bool hash, bool transactionData)
             }
         }
 
+        /// <summary>
+        /// Executes every <see cref="IAction"/> in the
+        /// <see cref="Transactions"/> step by step, and emits a quadruple of
+        /// a transaction, an action, input context, and output states
+        /// for each step.
+        /// <para>If the needed values are only the final states of each
+        /// transaction, use <see cref="EvaluateActions"/> method instead.
+        /// </para>
+        /// </summary>
+        /// <param name="accountStateGetter">An <see cref="AccountStateGetter"/>
+        /// delegate to get a previous state.
+        /// A <c>null</c> value, which is default, means a constant function
+        /// that returns <c>null</c>.</param>
+        /// <returns>Enumerates quadruples of a transaction, an action,
+        /// input context, and output states for each action.
+        /// The order of quadruples are the same to
+        /// the <see cref="Transactions"/> and their
+        /// <see cref="Transaction{T}.Actions"/> (e.g., tx&#xb9;-act&#xb9;,
+        /// tx&#xb9;-act&#xb2;, tx&#xb2;-act&#xb9;, tx&#xb2;-act&#xb2;,
+        /// &#x2026;).
+        /// Note that each <see cref="IActionContext.Random"/> object has
+        /// a unconsumed state.
+        /// </returns>
+        [Pure]
+        public
+        IEnumerable<(Transaction<T>, T, IActionContext, IAccountStateDelta)>
```

Comment:
Quadruples seems to be a bit complicated as a return type for public methods. How about creating a type?

Reply:
Fixed: 19089e75db6287a8e9de6eb47b92f5d72eef0d1d.